aos-rcar-gen4.yaml

```diff
@@ -134,6 +134,8 @@ components:
         # Do not install kernel image to rootfs to decrease initrd size
         - ["RDEPENDS_${KERNEL_PACKAGE_NAME}-base", ""]
 
+        # Add runX to the system
+        - [IMAGE_INSTALL_append, " runx"]
```

Comment:
Can you add it into https://github.com/aoscloud/meta-aos-rcar-gen4/blob/main/meta-aos-rcar-gen4-dom0/recipes-core/core-image-thin-initramfs/core-image-thin-initramfs.bbappend ?

Reply:
I've added this here because of the following advantages:
- IMAGE_INSTALL is related only for the image. That's why there is no risk of affecting another recipes
- We don't need to change recipe when migrating for example from core-image-thin-initramfs to core-image-minimal
- We have the whole set of tools in the same place.
- This was the main idea of the moulin. 
Maybe I've missed something? Or I can make separate variable for installation and use it from core-image-thin-initramfs as option. What do you think?

Reply:
ok